src/hooks/useUniswapMarketDetails.ts

```diff
@@ -99,7 +135,8 @@ export default function useUniswapMarketDetails({
         updateInputAmount(undefined, undefined, false);
         setIsSufficientBalance(true);
       } else {
-        const rawUpdatedInputAmount = tradeDetails?.inputAmount?.toExact();
+        if (!tradeDetails) return;
+        const rawUpdatedInputAmount = tradeDetails.inputAmount.toExact();
```

Comment:
Is it worth to keep the optional in `inputAmount?.toExact() ` ?

Reply:
Done!